### ISSUE TITLE
[BUG] Unable to disable --use-urandom in step1 signlenode python

### ISSUE THREAD

**Issue by brontide** [EMCECS/ECS-CommunityEdition#170]
``` python
    parser.add_argument('--use-urandom', dest='use_urandom', action='store_true', default=True,
                        help='If present, /dev/random will be mapped to /dev/urandom on the host.  If you container starts up slow the first time could help.',
                        required=False)
```

Having it as action="store_true" and default=True mean that it's always enabled.  I had to set default=False to turn it off.

**Comment by padthaitofuhot** [comment]
@brontide Thanks for bringing this to our attention.  
@jasoncwik I'll check to see if this was changed in the legacy scripts or not.  If not, it should be an easy fix.

I need to see what the disposition of random devices are in the new installer.  We may need to implement random/urandom remapping.

**Comment by padthaitofuhot** [comment]
Ok, PR on deck for legacy scripts, and I've added this issue to the Installer 2.0 project.

**Comment by padthaitofuhot** [comment]
Closed by #234
